client/components/site-icon/style.scss

```diff
@@ -9,13 +5,13 @@
 	overflow: hidden;
 	align-self: center;
 	margin: 0;
+	text-align: center;
 
 	// Globe icon for sites without an icon
 	&.is-blank {
-		.noticon {
+		.gridicon {
 			color: $white;
-			text-align: center;
-			z-index: z-index( 'root', '.site-icon.is-blank .noticon' );
+			z-index: z-index( 'root', '.site-icon.is-blank .gridicon' );
```

Comment:
You may find https://github.com/gwwar/z-context useful, though the extension is still a bit rough.